packages/opub-ui/src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Toast } from './Toast';
+import { Button } from '../Button';
+import { useState } from 'react';
+
+/**
+ * Toast Description
```

Comment:
Add description and reference link to the story

Reply:
Added stories with description